Common/src/main/resources/assets/advancednetherite/lang/fr_fr.json

```diff
@@ -0,0 +1,102 @@
+  "item.advancednetherite.netherite_gold_chestplate": "Plastron en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_chestplate": "Plastron en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_chestplate": "Plastron en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_leggings": "Jambières en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_leggings": "Jambières en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_leggings": "Jambières en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_leggings": "Jambières en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_boots": "Bottes en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_boots": "Bottes en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_boots": "Bottes en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_boots": "Bottes en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_axe": "Hache en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_axe": "Hache en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_axe": "Hache en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_axe": "Hache en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_hoe": "Houe en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_hoe": "Houe en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_hoe": "Houe en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_hoe": "Houe en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_pickaxe": "Pioche en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_pickaxe": "Pioche en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_pickaxe": "Pioche en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_pickaxe": "Pioche en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_shovel": "Pelle en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_shovel": "Pelle en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_shovel": "Pelle en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_shovel": "Pelle en Netherite-Diamant",
+
+  "item.advancednetherite.netherite_iron_sword": "Épée en Netherite-Fer",
+  "item.advancednetherite.netherite_gold_sword": "Épée en Netherite-Or",
+  "item.advancednetherite.netherite_emerald_sword": "Épée en Netherite-Émeraude",
+  "item.advancednetherite.netherite_diamond_sword": "Épée en Netherite-Diamant",
+
+  "itemGroup.advancednetherite.tab": "Advanced Netherite",
+  
+  "tooltip.advancednetherite.armor.enderman_passive": "+ Enderman pacifique",
+  "tooltip.advancednetherite.armor.piglin_passive": "+ Piglins pacifique",
+  "tooltip.advancednetherite.armor.phantom_passive": "+ Phantom pacifique",
+
+  "tooltip.advancednetherite.hoe.additional_crop_drops": "+ Chance de récolter des récoltes supplémentaires lors de la récolte de cultures mûres",
```

Comment:
Maybe change into
"+ Chance d'obtenir des récoltes supplémentaires lors de la récolte de cultures"

Reply:
I'll leave that up to you to decide, as I'm not familiar with the french language myself.